src/quo2/components/colors/color_picker/view.cljs

```diff
@@ -61,17 +19,19 @@
    - `blur?` Boolean to enable blur background support.}"
   [{:keys [default-selected?]}]
   (let [internal-selected (reagent/atom default-selected?)]
-    (fn [{:keys [blur? on-change selected]}]
+    (fn [{:keys [blur? on-change selected feng-shui?]}]
       (when (and (not (nil? selected)) (not= @internal-selected selected))
         (reset! internal-selected selected))
-      [rn/view {:style style/color-picker-container}
-       (doall (map-indexed (fn [index color]
-                             [:<> {:key (color :name)}
-                              [color-item
-                               (merge color
-                                      {:selected? (= (color :name) @internal-selected)
-                                       :on-press  #(on-change-handler internal-selected % on-change)
-                                       :blur?     blur?})]
-                              (when (= index 5) [rn/view {:style style/flex-break}])])
-                           (picker-colors blur?)))])))
-
+      [rn/scroll-view
+       {:horizontal                        true
+        :shows-horizontal-scroll-indicator false}
+       (doall (map (fn [color]
+                     [color/view
+                      {:selected? (= color @internal-selected)
+                       :on-press  #(on-change-handler internal-selected % on-change)
+                       :blur?     blur?
+                       :key       color
+                       :color     color}])
+                   ;; TODO: using :feng-shui? temporarily while b & w is being developed.
+                   ;; https://github.com/status-im/status-mobile/issues/15442
```

Comment:
The issue has been closed. Why is it still being referenced here?

Reply:
It was already referenced in code, i am not sure what is the current state of that. Do you know @J-Son89 ?

Reply:
maybe @Parveshdhull knows?

Reply:
Side note: that's also one of the reasons we have in the guidelines to not add TODO comments without our username attached, so others can quickly ping us or whomever wrote the comment https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#using-todos-comments

Reply:
The issues was moved to dicussions. I have linked the dicussion instead: https://github.com/status-im/status-mobile/discussions/16676